ct.yaml

```diff
@@ -12,3 +12,6 @@ chart-repos:
   - minio=https://helm.min.io
 helm-extra-args: --timeout 600s
 validate-maintainers: false
+# TODO: Remove this after the Helm chart for agent-operator is merged (PR #599)
+excluded-charts:
+  - agent
```

Comment:
Why do we need it?

Reply:
This is here to skip running tests for this chart because it contains `GrafanaAgent` Custom Resource whose CRD is deployed by the [`agent-operator` chart](https://github.com/grafana/helm-charts/pull/599) that is not yet merged. Once the `agent-operator` chart is merged, we should remove this exclusion and perhaps define dependency on the `agent-operator` chart.